packaging/Shell/install-fedora-deps.sh

```diff
@@ -56,22 +56,22 @@ sudo ldconfig
 
 # Install CIMpp from source
 cd /tmp && \
-	git clone --recurse-submodules --depth 1 https://github.com/cim-iec/libcimpp.git && \
-	mkdir -p libcimpp/build && cd libcimpp/build && \
-	cmake ${CMAKE_OPTS} .. \
+    git clone --recurse-submodules --depth 1 https://github.com/cim-iec/libcimpp.git && \
+    mkdir -p libcimpp/build && cd libcimpp/build && \
+    cmake ${CMAKE_OPTS} .. \
         -DCMAKE_INSTALL_LIBDIR=/usr/local/lib64 \
         -DUSE_CIM_VERSION=${CIM_VERSION} \
         -DBUILD_SHARED_LIBS=ON \
         -DBUILD_ARABICA_EXAMPLES=OFF && \
     make ${MAKE_OPTS} install && \
-	rm -rf /tmp/libcimpp
+    rm -rf /tmp/libcimpp
 
 # Install VILLASnode from source
```

Comment:
I think it may be the case that L36 already installed `villas-node `as dependency due to it being listed in the `.toml`, this may be also the case for the similar shell files and even the dockerfiles

Reply:
L36 only installed the [VILLASnode Python package](https://pypi.org/project/villas-node/) but noch actually compile/build VILLASnode's C++ code

Reply:
Thanks! Then disregard this too